server/cpho/views/upload_indicators.py

```diff
@@ -203,6 +254,38 @@ def clean_csv_file(self):
 
             # data_dimension = deduce_dimension_type(data_row, idx)
             # print(data_dimension)
+
+            # checking if indicator already exists
+            indicator_obj = Indicator.objects.filter(
+                name=data_row["Indicator"],
+                category=mapper["category_mapper"][data_row["Category"]],
+                sub_category=mapper["subcategory_mapper"][data_row["Topic"]],
+                detailed_indicator=data_row["Detailed Indicator"],
+                sub_indicator_measurement=data_row[
+                    "Sub_Indicator_Measurement"
+                ],
+            )
```

Comment:
Rather than testing the length twice and calling `[0]` (3 queries), just call `indicator_obj = Indicator.objects.filter(...).first()` and test for `None` rather than the length (1 query)